bootstrap-v5.sh

```diff
@@ -37,6 +37,8 @@ function set_internal_variables {
 
     GIT_REPO=https://github.com/ausaccessfed/shibboleth-idp5-installer.git
     GIT_BRANCH=main
```

Comment:
Is this redundant now? As we have a new declaration below:     GIT_BRANCH=feature-metadata-driven

Reply:
Oops, though I had removed the second one. It needs to be GIT_BRANCH=main.